src/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -66,6 +67,7 @@ protected NuGetPowerShellBaseCommand()
             SourceControlManagerProvider = ServiceLocator.GetInstance<ISourceControlManagerProvider>();
             _commonOperations = ServiceLocator.GetInstance<ICommonOperations>();
             PackageRestoreManager = ServiceLocator.GetInstance<IPackageRestoreManager>();
+            _deleteOnRestartManager = ServiceLocator.GetInstance<IDeleteOnRestartManager>();
```

Comment:
Not sure how we missed this in the review. You already have an instance of this _deleteOnRestartManager on the base class for all cmdlets, which is, NuGetPowershellBaseCommand. Just make this Property protected or something and use it from UninstallPackageCommand and PackageActionBaseCommand class. Why do we have the same property on several classes?